go.mod

```diff
@@ -1,11 +1,13 @@
 module github.com/runfinch/finch
 
-go 1.22.0
+go 1.22.7
+
+toolchain go1.22.10
```

Comment:
Any particular reason we are adding the toolchain directive here? It seems unrelated to the change so I wonder if this warrants a separate PR/discussion.

Not particularly opinionated on it, just found it a little strange, feel free to just resolve.

Reply:
I believe newer versions of go automatically add it. I can remove it though